awx/ui_next/src/util/dates.test.jsx

```diff
@@ -0,0 +1,12 @@
+import { formatDateString } from './dates';
+
+describe('formatDateString', () => {
+  test('it returns the expected value', () => {
+    expect(formatDateString('')).toEqual('Invalid Date');
+    expect(formatDateString({})).toEqual('Invalid Date');
+    expect(formatDateString(undefined)).toEqual('Invalid Date');
+    expect(formatDateString('2018-01-31T01:14:52.969227Z')).toEqual(
```

Comment:
this test implies the browser navigator will be a specific value...I feel like it might make sense to hardcode this param to make the test deterministic.